assets/templates/related-list-pages.tmpl

```diff
@@ -1,27 +1,31 @@
-{{ $lang := .Language }}
 <div class="ons-container search__container">
   <div class="ons-grid">
-
     <div class="ons-grid__col ons-col-12@m">
-        <section class="search__summary" role="contentinfo">
-            {{ template "partials/type-label" . }}
-            {{ template "partials/related-list-pages-title" . }}
-            <p> {{ .Metadata.Description }}</p>
+      {{ if gt (len .Error.ErrorItems) 0 }}
+        <div class="ons-u-mb-l">
+          {{ template "partials/error-summary" .Error }}
+        </div>
+      {{ else }}
+        <section
+          class="search__summary"
+          role="contentinfo"
+        >
+          {{ template "partials/type-label" . }}
+          {{ template "partials/related-list-pages-title" . }}
+          <p>{{ .Metadata.Description }}</p>
```

Comment:
we're removing the whole page if there's errors?

Reply:
yep the partials require some mapped data (search response I believe) that is causing the app to OOM as the mapping fails (especially for the higher page params i.e 100) and the app doesn't know how to handle and by design we just let the app OOM 😄